addons/binding/org.openhab.binding.harmonyhub/.classpath

```diff
@@ -16,5 +16,6 @@
 	<classpathentry kind="lib" path="lib/smack-tcp-4.2.0.jar"/>
 	<classpathentry kind="lib" path="lib/xpp3-1.1.4c.jar"/>
 	<classpathentry kind="lib" path="lib/harmony-java-client-1.2.2.jar"/>
```

Comment:
Can you also remove the old and removed JARs from this `.classpath` file so Eclipse doesn't show errors?

Reply:
👍